cpp/include/cudf/table/experimental/row_operators.cuh

```diff
@@ -381,7 +381,7 @@ class device_row_comparator {
 
     template <typename Element,
               CUDF_ENABLE_IF(has_nested_columns and std::is_same_v<Element, cudf::struct_view>)>
-    __device__ cuda::std::pair<weak_ordering, int> operator()(
+    __attribute__((noinline)) __device__ cuda::std::pair<weak_ordering, int> operator()(
```

Comment:
I think we typically write this as `__noinline__`. Internally, I believe CUDA expands it to the `__attribute__` form. https://docs.nvidia.com/cuda/cuda-c-programming-guide/#noinline-and-forceinline